packages/vm/src/modules/in_memory_cache.rs

```diff
@@ -1,9 +1,16 @@
 use clru::CLruCache;
 use wasmer::Module;
 
-use crate::{Checksum, Size, VmResult};
+use crate::{Checksum, Size, VmError, VmResult};
 
-const ESTIMATED_MODULE_SIZE: Size = Size::mebi(10);
+// Minimum module size, in Megabytes.
```

Comment:
```suggestion
// Minimum module size.
```

Otherwise this would be 250*1024 megabytes